### PR TITLE
English version added

### PR DESCRIPTION
**Pull request by ewolff**
Here is a PR to add the English version from the Word doc to the ASCIIdoc document.
@programming-wolf You wanted to review and advice - so that is why I requested a review.

### CODE REVIEW

File: docs/00b-basics/01-what-to-expect-of-this-module.adoc [hunk 1]
```diff
@@ -11,8 +11,14 @@ Am Ende des Moduls kennen die Teilnehmerinnen und Teilnehmer Grundlagen von Micr
 // tag::EN[]
 === What does the module “{curriculum-short}” convey?
 
-The module presents {curriculum-short} to the participants …
-At the end of the module, the participants know … and are able to …
+The module shows {curriculum-short} the participants how modern
```
Comment (programming-wolf) [inline docs/00b-basics/01-what-to-expect-of-this-module.adoc]: `The module {curriculum-short} shows ...`

File: docs/00b-basics/02-curriculum-structure-and-chronological-breakdown.adoc [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
 | 1. Motivation | 120
-| 2. Modularisierung | 240
-| 3. Integration | 150
-| 4. Installation und Roll Out | 150
-| 5. Betrieb, Überwachung und Fehleranalyse | 150
+| 2. Modularisierung | 150
+| 3. Integration | 120
+| 4. Installation und Roll Out | 120
+| 5. Betrieb, Überwachung und Fehleranalyse | 120
 | 6. Case Study | 150
```
Comment (programming-wolf) [inline docs/00b-basics/02-curriculum-structure-and-chronological-breakdown.adoc:12]: Is there no update to the English part of this file? Or is the English version already correct?

Comment (programming-wolf) [reply docs/00b-basics/02-curriculum-structure-and-chronological-breakdown.adoc:12]: Ah, forget that, I looked at the file.

File: docs/01-motivation/02-learning-goals.adoc [hunk 3]
```diff
@@ -33,7 +33,7 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
 [[LZ-1-5]]
 ==== LZ 1-5: Themen und Buzzwords einordnen
   * Gesetz von Conway
-  * Partitionierbarkeit als Qualitätsmerkmal
+  * Starke Isolation gegen Ausfälle, bei Deployments usw  als Qualitätsmerkmal
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc]: `Starke Isolation gegen Ausfälle bei Deployments usw. als Qualitätsmerkmal`

File: docs/01-motivation/02-learning-goals.adoc [hunk 4]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
 
 // tag::EN[]
 [[LG-1-1]]
-==== LG 1-1: The is the first learning goal, in category xy
-tbd.
+==== LG 1-1: Quality Goal for Flexible Architectures
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc]: `Quality Goals`

File: docs/01-motivation/02-learning-goals.adoc [hunk 5]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
...
+Architectures can be optimised for different quality goals. In this
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc]: Here you use `optimised` (British English), the next learning goal uses `organization` (American English). 
iSAQB uses American English across all the documents, so you should check the files for occurrences of British English and adjust them accordingly.

File: docs/01-motivation/02-learning-goals.adoc [hunk 6]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
...
+module, participants learn how to create flexible architectures that
+allow rapid deployment and thus rapid feedback from the application of
+the system.
+
+[[LG-1-2]]
+==== LG 1-2: Interplay Between Architecture Types and Organization
+They have understood the drivers for the architecture types taught in
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc:53]: Who are "they"?

File: docs/01-motivation/02-learning-goals.adoc [hunk 7]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
...
+this curriculum module, the implications of these drivers for the
+architectures, and the interaction of the architectures with the
+organisation, processes, and technologies.
+
+[[LG-1-3]]
+==== LG 1-3: Community and Apply Trade-Offs between Architecture Types
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc:59]: Community? I guess you mean `Communicate`?

File: docs/01-motivation/02-learning-goals.adoc [hunk 8]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
...
+They have understood the trade-offs of the architecture types
+presented (at least microservices, Self-Contained Systems and
+Deployment Monoliths) and are able to communicate them as well as
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc]: `at least microservices, self-contained systems and deployment monoliths` 

There's no need to capitalize some of the variants.

File: docs/01-motivation/02-learning-goals.adoc [hunk 9]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
...
+* Unification of the different environments (e. g., development, test,
+  QA, production) reduces the risk of lately detected and (in other
+  environments) non-reproducible errors due to different
+  configurations.
+* Unification and automation are essential aspects of continuous
+  delivery.
+* Continuous integration is a prerequisite for continuous delivery.
+* A suitable architecture is the prerequisite for the parallelisation
+  of the development as well as the independent commissioning of
+  independent modules. This can, but do not have to be "services".
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc:88]: `These can, but do not have to be, "services".`

File: docs/01-motivation/02-learning-goals.adoc [hunk 10]
```diff
@@ -42,8 +42,71 @@ Sie haben die Tradeoffs der vorgestellten Architektur-Typen (mindestens Microser
...
+* Certain types of isolation are much easier to implement between
+  processes with remote communication.
+* Remote communication, however, has disadvantages - for example many
+  new sources of errors.
+
+
+[[LG-1-5]]
+==== LG 1-5: Understanding Buzzwords 
+* Conway’s law
+* Strong isolation concerning crashes, deployment etc as a quality feature
```
Comment (programming-wolf) [inline docs/01-motivation/02-learning-goals.adoc:104]: `etc.`

File: docs/02-modularization/02-learning-goals.adoc [hunk 11]
```diff
@@ -77,12 +77,146 @@
 
 // tag::EN[]
 [[LG-2-1]]
-==== LG 2-1: TBD
-tbd.
+==== LG 2-1: Decomposition into Block
```
Comment (programming-wolf) [inline docs/02-modularization/02-learning-goals.adoc:80]: `Blocks`

File: docs/02-modularization/02-learning-goals.adoc [hunk 12]
```diff
@@ -77,12 +77,146 @@
...
+.What shall participants be capable of?
+- The participants should be able to evaluate and select technical
+  modularisation concepts in a project-specific manner.
+- The participants should be able to illustrate and analyse the
+  relationships between modules as well as between modules and
+  subdomains (context mapping).
+- Participants can develop a concept to build a system of services.
+
+.What should participants understand?
+- Modularisation helps to achieve goals such as parallelisation of
```
Comment (programming-wolf) [inline docs/02-modularization/02-learning-goals.adoc]: Sometimes you use `-` to start bullet lists, sometimes you use `*`. One of them should be used consistently, I recommend `*`.

File: docs/02-modularization/02-learning-goals.adoc [hunk 13]
```diff
@@ -77,12 +77,146 @@
...
+- Participants understand that, when distributing to different
+  processes, some dependencies no longer exist in the implementation,
+  but rather arise at runtime. This increases the requirements for
+  monitoring these interfaces.
+- Microservices are independent deployment units and therefore
+  independent processes that expose their functions through
+  lightweight protocols, but may also have a UI. Different technology
+  decisions can be made for the implementation of each individual
+  microservice.
+- A Self-Contained System (SCS) is a functionally independent
```
Comment (programming-wolf) [inline docs/02-modularization/02-learning-goals.adoc]: As long as it is not used in a header, you should write it in lowercase: self-contained system.

File: docs/02-modularization/02-learning-goals.adoc [hunk 14]
```diff
@@ -77,12 +77,146 @@
...
+  system. It usually includes UI and persistence. It may consist of
+  several Microservices. Usually a SCS covers a functional bounded
```
Comment (programming-wolf) [inline docs/02-modularization/02-learning-goals.adoc]: `microservices`

File: docs/02-modularization/02-learning-goals.adoc [hunk 15]
```diff
@@ -77,12 +77,146 @@
...
+  applications, processes, microservices, SCS.
+- The participants should know "The Twelve-Factor App".
+
+
+
+[[LG-2-3]]
+==== LG 2-3: Strategies for Modularization
+.What shall participants be capable of?
+- Participants can evaluate the consequences of different
+  modularisation strategies and compare the efforts of the
```
Comment (programming-wolf) [inline docs/02-modularization/02-learning-goals.adoc]: You use American English in the header, and then change to British English. 😺

File: docs/02-modularization/02-learning-goals.adoc [hunk 16]
```diff
@@ -77,12 +77,146 @@
...
+  modularisation with the expected benefits.
+- Participants can assess the impact of the modularisation strategy on
+  the autonomy of building blocks at development time and at run time.
+- The participants can choose a suitable modularisation as well as a
+  suitable granularity of the modularisation - depending on the
```
Comment (programming-wolf) [inline docs/02-modularization/02-learning-goals.adoc]: If you want to use a dash in English, you have to use the longest of them: `--`

File: docs/03-integration/01-duration-terms.adoc [hunk 17]
```diff
@@ -10,10 +10,10 @@ Frontend Integration, Legacy Systeme, Authentifizierung, Autorisierung, (lose) K
 
 // tag::EN[]
 |===
-| Duration: 120 min | Practice time: 30 min
+| Duration: 90 min | Practice time: 30 min
 |===
 
 === Terms and Principles
-Term 1, Term 2, Term 3
+Frontend integration, legacy systems, authentication, authorisation, (loose) coupling, scala-bility, messaging patterns, domain events, decentralised data management.
```
Comment (programming-wolf) [inline docs/03-integration/01-duration-terms.adoc]: `scalability`

File: docs/03-integration/02-learning-goals.adoc [hunk 18]
```diff
@@ -44,12 +44,89 @@
...
+==== LG 3-1: Integration Technologies
+- The participants should choose an integration strategy that best
+  suits the particular problem. This can be, for example, front-end
+  integration, integration via RPC mechanisms, message-oriented
+  middleware, REST, or replication of data.
+- The participants can propose a suitable integration depending on the
+  quality goals and the knowledge of the team.
+- The participants should know the benefits and drawbacks of different
+  integration mechanisms. These include front-end integration with
+  Mash Ups, Middle Tier integration, and integration through
```
Comment (programming-wolf) [inline docs/03-integration/02-learning-goals.adoc]: `mash-ups` (I guess)
`middle tier`

File: docs/03-integration/02-learning-goals.adoc [hunk 19]
```diff
@@ -44,12 +44,89 @@
...
+- An integration via data enables high autonomy, nevertheless it is
+  bought by the necessity for redundant data storage and the
+  necessary synchronisation. It must not be assumed that other systems
+  use the same schemes, because this prevents an independent
+  development of the schemata. Therefore, an adequate transformation
+  has to be provided for the integration.
+- Participants should know approaches for front-end integration.
+- Participants should know Technologies for the integration of
+  services: REST, RPC, message-oriented middleware.
+- Participants should know Challenges of the usage of shared data.
```
Comment (programming-wolf) [inline docs/03-integration/02-learning-goals.adoc]: `challenges`, `database`

You should check Capitalization in all the files and convert (most of) the words to lowercase.

File: docs/04-rollout/02-learning-goals.adoc [hunk 20]
```diff
@@ -34,13 +34,61 @@
 
 // tag::EN[]
 [[LG-4-1]]
-==== LG 4-1: TBD
-tbd.
+==== LG 4-1: Aspectrs for Rollout and Continuous Delivery
```
Comment (programming-wolf) [inline docs/04-rollout/02-learning-goals.adoc:37]: `Aspects`

File: docs/05-operations/01-duration-terms.adoc [hunk 21]
```diff
@@ -14,7 +14,7 @@ Monitoring, Operations, Logging, Tracing, Metrics, Alerting.
 |===
 
 === Terms and Principles
-Term 1, Term 2, Term 3
+Monitoring, Operations, Logging, Tracing, Metrics.
```
Comment (programming-wolf) [inline docs/05-operations/01-duration-terms.adoc]: German text also has "Alerting". Not relevant in English, or not relevant in general?

File: docs/05-operations/02-learning-goals.adoc [hunk 22]
```diff
@@ -44,12 +44,66 @@
 
 // tag::EN[]
 [[LG-5-1]]
-==== LG 5-1: TBD
-tbd.
+==== LG 5-1: Analyzing Error in Distributed Systems
```
Comment (programming-wolf) [inline docs/05-operations/02-learning-goals.adoc]: `Errors`

File: docs/05-operations/02-learning-goals.adoc [hunk 23]
```diff
@@ -44,12 +44,66 @@
...
+  * The participants should understand which information can be
+    obtained from log data and which may (better) be obtained by
+    instrumentation of the code with metric probes.
 
 [[LG-5-2]]
-==== LG 5-2: TBD
-tbd.
+==== LG 5-2: Difference between Metrics, Logs, and Traces
+.What should participants understand?
+  *	Logging and monitoring can include both functional as well as
```
Comment (programming-wolf) [inline docs/05-operations/02-learning-goals.adoc]: Unnecessary whitespace

File: docs/05-operations/02-learning-goals.adoc [hunk 24]
```diff
@@ -44,12 +44,66 @@
...
+  * Tools for centralised log data management
+  * Tools for centralised metrics processing
+  * Difference between business, application and system metrics
+  * The meaning of important, system-independent application and system metrics
+
+[[LG-5-3]] 
+==== LG 5-3: Different Operation Models
+.What shall participants be capable of?
+  * Deciding to use a centralized or decentralized operations model
+  depending on the context - like organization and skill sets.
```
Comment (programming-wolf) [inline docs/05-operations/02-learning-goals.adoc]: Wrong dash

File: docs/07-outlook/02-learning-goals.adoc [hunk 25]
```diff
@@ -25,20 +24,62 @@
...
 // tag::EN[]
 [[LG-7-1]]
 ==== LG 7-1: Consistency Models
-tbd.
+- What shall participants be capable of?
+  * The participants should know various consistency models. They
+    should basically know the trade-offs of the different consistency
+    models.
+- What should participants understand?
+  *	The need for ACID transactions is much lower than is often
```
Comment (programming-wolf) [inline docs/07-outlook/02-learning-goals.adoc]: Whitespace

File: docs/07-outlook/02-learning-goals.adoc [hunk 26]
```diff
@@ -25,20 +24,62 @@
...
+  * Isolation und Latenzüberwachung als sinnvolle Einstiegsprinzipien von Resilient Software Design
   * Grundlegende Resilience-Muster wie Bulkhead, Circuit Breaker, Redundanz, Failover
 
 // end::DE[]
 
 // tag::EN[]
 [[LG-7-1]]
 ==== LG 7-1: Consistency Models
-tbd.
+- What shall participants be capable of?
```
Comment (programming-wolf) [inline docs/07-outlook/02-learning-goals.adoc]: Sometimes you use bullets, sometimes you use
`.What shall participants...`

Should be the same everywhere

**Review by programming-wolf — CHANGES_REQUESTED** [review]
Second review required. 🙂

File: docs/00b-basics/01-what-to-expect-of-this-module.adoc (hunk 1 above)
Comment (ewolff) [reply docs/00b-basics/01-what-to-expect-of-this-module.adoc]: fixed, thanks

File: docs/01-motivation/02-learning-goals.adoc (hunk 3 above)
Comment (ewolff) [reply docs/01-motivation/02-learning-goals.adoc]: Die Deployments sind isoliert: Ein Microservice kann unabhängig von einem anderen deployt werden.

Comment (ewolff) [reply docs/01-motivation/02-learning-goals.adoc]: Umformuliert. Danke.

File: docs/01-motivation/02-learning-goals.adoc (hunk 4 above)
Comment (ewolff) [reply docs/01-motivation/02-learning-goals.adoc]: Fixed. Thanks.

File: docs/01-motivation/02-learning-goals.adoc (hunk 5 above)
Comment (ewolff) [reply docs/01-motivation/02-learning-goals.adoc]: Actually the original translation as provided by iSAQB used British English.

File: docs/05-operations/02-learning-goals.adoc (hunk 22 above)
Comment (ewolff) [reply docs/05-operations/02-learning-goals.adoc]: Fixed. Thanks

File: docs/05-operations/02-learning-goals.adoc (hunk 23 above)
Comment (ewolff) [reply docs/05-operations/02-learning-goals.adoc]: Fixed. Thanks.

File: docs/05-operations/02-learning-goals.adoc (hunk 24 above)
Comment (ewolff) [reply docs/05-operations/02-learning-goals.adoc]: Fixed. Thanks

File: docs/07-outlook/02-learning-goals.adoc (hunk 25 above)
Comment (ewolff) [reply docs/07-outlook/02-learning-goals.adoc]: Fixed. Thanks.

File: docs/07-outlook/02-learning-goals.adoc (hunk 26 above)
Comment (ewolff) [reply docs/07-outlook/02-learning-goals.adoc]: This will be fixed later. I need to review the whole goals anyway. I will unify them, too.

Comment (ewolff) [reply docs/07-outlook/02-learning-goals.adoc]: Actually I just fixed it. Was rather minor.

File: docs/01-motivation/02-learning-goals.adoc (hunk 8 above)
Comment (aheusingfeld) [reply docs/01-motivation/02-learning-goals.adoc]: IMPOV it‘s common practice to capitalize „buzzwords“ i.e. well-known terms. Yet, we should be consistent and also capitalize Microservices.

File: docs/01-motivation/02-learning-goals.adoc (hunk 9 above)
Comment (aheusingfeld) [reply docs/01-motivation/02-learning-goals.adoc:88]: Either there should be a „be“ with „This can _be_“ or the comma needs to be before the „be“ i.e. „This can - but do not have to - be services“

File: docs/01-motivation/02-learning-goals.adoc (hunk 8 above)
Comment (ewolff) [reply docs/01-motivation/02-learning-goals.adoc]: Having written a few books about microservices, I can assure you it's written with a lowercase m.
